types/graphlib-dot/index.d.ts

```diff
@@ -0,0 +1,24 @@
+// Type definitions for graphlib-dot 0.6
+// Project: https://github.com/dagrejs/graphlib-dot
+// Definitions by: Dom Parfitt <https://github.com/DomParfitt>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+import { Graph } from 'graphlib';
```

Comment:
According to https://github.com/dagrejs/graphlib-dot/wiki#graphlibdot-graphlib the graphlibDot is available as browser bundle and hence needs to add global definitions as well. For more information, please refer to https://github.com/DefinitelyTyped/DefinitelyTyped#how-do-i-write-definitions-for-packages-that-can-be-used-globally-and-as-a-module

Reply:
Thanks for the help. Will do.

Reply:
You also need YourLibraryName-global.test.ts